### PR TITLE
Adding Cyren Inbox Security cortex integration pack

### PR DESCRIPTION
**Pull request by eran-levy**
## Description
This integration imports incidents from Cyren Inbox Security into XSOAR and includes a layout and  playbook to aid incident resolution.

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by eran-levy** [comment]
Signed the CLA - Contributor License Agreement - with a wrong name - seems like we should sign with the github username eran-levy instead of Eran Levy to recognize that. Im not able to edit the signed agreement can you help me please?

File: Packs/CyrenInboxSecurity/Playbooks/Cyren_Inbox_Security_playbook.yml [hunk 1]
```diff
@@ -0,0 +1,752 @@
+description: 'Processes Cyren Incidents, sets resolutions, and applies remediations
+  to end-user mailboxes. '
+id: be5080ec-ac04-459c-8eb0-0f0323c7af50
```
Comment (evisochek) [inline Packs/CyrenInboxSecurity/Playbooks/Cyren_Inbox_Security_playbook.yml:3]: The ID and the name should we the same, please replace the ID with the value as the name

**Comment by ShahafBenYakir** [comment]
@eran-levy The reason is because you committed the code using different username than your local git user. If you have the creds for the other user please connect to Github using this user-name. See [this](https://xsoar.pan.dev/docs/concepts/faq) for further info.

**Comment by yaakovpraisler** [comment]
@eran-levy - Please register your contribution by filling the [Contribution Registration](https://forms.gle/XDfxU4E61ZwEESSMA) form, so I can start review.

**Comment by eran-levy** [comment]
created a new PR #14074

**Pull request closed without merging**